posthog/models/cohort/util.py

```diff
@@ -362,3 +362,24 @@ def get_all_cohort_ids_by_person_uuid(uuid: str, team_id: int) -> List[int]:
     cohort_ids = _get_cohort_ids_by_person_uuid(uuid, team_id)
     static_cohort_ids = _get_static_cohort_ids_by_person_uuid(uuid, team_id)
     return [*cohort_ids, *static_cohort_ids]
+
+
+def get_dependent_cohorts(cohort: Cohort) -> List[Cohort]:
```

Comment:
i assume this is what's checking to see if there's nested behavioral cohorts in a cohort

Reply:
correct